tests/test_starlette.py

```diff
@@ -74,6 +78,8 @@ class User(db.Model):
 
     e = await gino.create_engine(PG_URL)
     try:
+        if factory:
```

Comment:
can we put it up to the previous `if factory` block?

Reply:
I think so - you may know better but I think I would have to move the `create_engine` statement up as well.

Reply:
No. The engine here is just to call `create_all` and not related to app initialization. As you can see, `e` is not passed to `init_app`

Reply:
Oh - wasn't sure if the init_app depended on having an engine. Thanks!